diff -up libktorrent-24.12.3/src/utp/connection.cpp.omv~ libktorrent-24.12.3/src/utp/connection.cpp
--- libktorrent-24.12.3/src/utp/connection.cpp.omv~	2025-03-21 19:06:33.911090493 +0100
+++ libktorrent-24.12.3/src/utp/connection.cpp	2025-03-21 19:06:40.327287846 +0100
@@ -13,6 +13,7 @@
 #include <QTextStream>
 #include <QThread>
 #include <time.h>
+#include <climits>
 #include <util/functions.h>
 #include <util/log.h>
 #include <util/sha1hash.h>
